GATK_QuickByte.md

```diff
@@ -320,35 +361,18 @@ Here is a sample PBS script combining everything we have above, with as much par
 
 	cat $src/sample_list | env_parallel -j $PBS_NUM_NODES --sshloginfile $PBS_NODEFILE \
 		'bwa mem \
-			-t $threads -M \
-			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSML{}" \
+			-t 8 -M \
+			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSM:{}" \
 			$reference \
 			$src/raw_reads/{}_R1.fastq.gz \
 			$src/raw_reads/{}_R2.fastq.gz \
-			> $src/alignments/{}.sam
+			> $src/sams/{}.sam
 		gatk MarkDuplicatesSpark \
```

Comment:
I have been having trouble with this line in the parallelized version. I keep getting this error: 

Caused by: htsjdk.samtools.SAMFormatException: Error parsing text SAM file. Not enough fields; Line unknown

Could this be due to the bwa mem -R flag?

Reply:
I'm pretty sure it's because the PBS script has the sam file in $src/sams, but they should be in $src/alignments. It seems to say that I changed it, but I'm not sure why I'd do that?

Reply:
I think I figured it out. Something is going wrong with bwa mem.

Reply:
I resolved it!